vars/mavenIntegrationTest.groovy

```diff
@@ -29,7 +29,7 @@ def call(body) {
         echo "WARNING: Integration tests are current DISABLED for these pipelines!"
 
     } else {
-        sh "mvn org.apache.maven.plugins:maven-failsafe-plugin:integration-test ${kubeNS} -P openshift-it -Dit.test=${config.itestPattern} -DfailIfNoTests=${config.failIfNoTests} org.apache.maven.plugins:maven-failsafe-plugin:verify"
+        sh "mvn org.apache.maven.plugins:maven-failsafe-plugin:integration-test ${kubeNS} -Dnamespace.use.current=false -DenableImageStreamDetection=true -P openshift-it -Dit.test=${config.itestPattern} -DfailIfNoTests=${config.failIfNoTests} org.apache.maven.plugins:maven-failsafe-plugin:verify"
```

Comment:
How about adding a comment explaining all parameters we are sending, just to understand in some months why we set them.

Reply:
I don't want these parameters to be here to start with - see my "Open points" section in the PR description.

Reply:
Yes I know but I put the comment just in case we decided for now to go with this approach.

Reply:
If we decide, I will :)